packages/wallets/src/features/cashier/modules/Transfer/components/TransferForm/TransferForm.scss

```diff
@@ -25,6 +25,7 @@
         padding: 0.8rem;
 
         @include mobile {
+            height: 9rem;
```

Comment:
No way to enforce this height implicitly by making the div fit content to have this height, example:

<img width="1373" alt="Screenshot 2024-01-24 at 17 44 08" src="https://github.com/binary-com/deriv-app/assets/119863957/8a14e761-837c-42d8-8fd2-fdc861bcef3e">

Reply:
hey @rostislav-deriv 
could we change height to min-height in order if `Select trading account or a Wallet` text will take 3 rows after translation instead of 2? and our height can increase

Reply:
@heorhi-deriv done

<img width="1482" alt="Screenshot 2024-01-25 at 11 11 22" src="https://github.com/binary-com/deriv-app/assets/119863957/46d957fb-b5ed-4415-8239-5ed0f1ca0560">